src/cmd-push-container

```diff
@@ -37,35 +37,43 @@ latest_build_path = f"builds/{latest_build}/{arch}"
 metapath = f"{latest_build_path}/meta.json"
 with open(metapath) as f:
     meta = json.load(f)
-ociarchive = os.path.join(latest_build_path, meta['images']['ostree']['path'])
 
-skopeoargs = ['skopeo', 'copy']
-if args.authfile is None:
-    args.authfile = os.environ.get("REGISTRY_AUTH_FILE")
-if args.authfile is not None:
-    skopeoargs.extend(['--authfile', args.authfile])
-if args.format is not None:
-    skopeoargs.extend(['--format', args.format])
-if ":" not in args.name:
-    container_name = args.name
-    # If a specific tag wasn't requested, add a default one
-    container_tag = f"{latest_build}-{arch}"
+extensions_ociarchive = os.path.join(latest_build_path, meta['images']['extensions-container']['path'])
+if not extensions_ociarchive:
+    ociarchives = ["ostree", "extensions-container"]
 else:
-    # Strip the tag out, as we will be injecting the digest below
-    # Note this implicitly errors out if there's more than one ':'
-    container_name, container_tag = args.name.rsplit(':')
-if args.tag_suffix:
-    container_tag = f"{container_tag}-{args.tag_suffix}"
-with tempfile.NamedTemporaryFile(dir='tmp', prefix='push-container-digestfile') as df:
-    skopeoargs.append(f"--digestfile={df.name}")
-    skopeoargs.extend([f"oci-archive:{ociarchive}", f"docker://{container_name}:{container_tag}"])
-    print(subprocess.list2cmdline(skopeoargs))
-    subprocess.check_call(skopeoargs)
-    df.seek(0)
-    digest = df.read().decode('utf-8').strip()
-    # Inject the oscontainer with SHA256 into the build metadata
-    meta['base-oscontainer'] = {'image': f"{container_name}@{digest}"}
-    metapath_new = f"{metapath}.new"
-    with open(metapath_new, 'w') as f:
-        json.dump(meta, f, sort_keys=True)
-    shutil.move(metapath_new, metapath)
+    ociarchives = ["ostree"]
+for archive_id in ociarchives:
```

Comment:
Ah, interesting.  I'd been thinking we'd have two separate calls to `push-container` in the pipeline, like `push-container --ostree` and `push-container --extensions` or something as separate calls.  But, I'm OK doing both at once too.

Reply:
Ahh I see the idea was since both need to be pushed together going forward then always check if the extensions container was there and push it. But changing it to a flag is a quick change.

Reply:
I'm definitely OK having one command push two containers; it becomes slightly odd since it's now really push-container**s** plural but...eh.  (Renaming it would require ratcheting into the pipelines, let's not do that)

Reply:
I'll change it. Thinking about it this might be bad if in the pipeline we are doing build and then push right away. Basically I guess that we would call this script twice either way. It will also minimize the changes on the code. No loops, etc.

Reply:
I guess if we don't want to adjust the pipelines then I added `--extensions` parameter only and then ostree is the default behavior.